src/style.css

```diff
@@ -1,3 +1,103 @@
-body {
+:root {
+  --main-color: #000;
+  --active-color: #0ad;
+  --background-color: #fff;
+  --font-size: 12px;
+  --line-height: 14px;
+  --font-family: "Roboto", "Helvetica", sans-serif;
+  --font-weight: 500;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+header,
+div,
+a,
+nav,
+ul,
+img {
   margin: 0;
+  padding: 0;
+}
+
+.body {
+  font-size: var(--font-size);
+  line-height: var(--line-height);
+  font-family: var(--font-family);
+  font-weight: var(--font-weight);
+  background-color: rgb(241, 231, 231);
+  color: var(--main-color);
+}
+
+.header {
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  text-transform: uppercase;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  background-color: var(--background-color);
+}
+
+.container {
+  margin: 0 50px;
+  width: 1200px;
+  height: 100%;
+  display: flex;
```

Comment:
1. `height` is redundant
2. `width` ruins the first requirement of the task
<img width="832" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/49764881/6617045c-b4cb-420a-8515-b91d02cbb2ec">